ccpp/data/GFS_typedefs.F90

```diff
@@ -6438,6 +6437,14 @@ subroutine interstitial_create (Interstitial, IM, Model)
        allocate (Interstitial%toa_src_sw           (IM,Model%rrtmgp_nGptsSW))
        allocate (Interstitial%toa_src_lw           (IM,Model%rrtmgp_nGptsLW))
        allocate (Interstitial%active_gases_array   (Model%nGases))
+       ! ty_gas_concs
```

Comment:
Obviously %ncol and %nlay are fixed throughout the run.

What about gas_name, concs, conc? Do they depend on each grid, and if yes, do they need to be persistent? If not, then we should add them to the %rad_reset calls for the DDT (for clarity) - in the next PR. If they need to be persistent, they need to be moved to radtend or something like that (also in the next PR).

Reply:
@climbfuji 
I set the ncol and nlay here once in the DDT, type-bound procedures rely on it later.
gas_concentrations(ty_gas_concs) should be added to rad_reset. I will make sure to include this in my next PR.